src/lava/proc/dense/process.py

```diff
@@ -47,22 +47,22 @@ class Dense(AbstractProcess):
     architectures.
```

Comment:
Deadlock avoidance in recurrent architectures is just one aspect. 
-> Circular buffer that stores output activations accumulated in current time step for future time steps.

Reply:
Changed the docstring to match the suggested description.